app/scripts/lib/experiments/base.js

```diff
@@ -107,7 +107,7 @@ function (Backbone, _, Storage, Url) {
         }
 
         if (_.isFunction(method)) {
```

Comment:
For a follow on PR, I'd like to make the mixin not only a view mixin, but a model mixin too, and mix it into this model.